### PR TITLE
updated loot.po

### PR DESCRIPTION
**Pull request by ZyPA**
Updated translations for danish

### CODE REVIEW

File: resources/l10n/da/LC_MESSAGES/loot.po [hunk 1]
```diff
@@ -4,1267 +4,2086 @@
...
+#: src/gui/qt/groups_editor/groups_editor_dialog.cpp:260
+msgid "Group already exists!"
+msgstr "Gruppe eksistere allerede!"
+
+#
+# File: src/gui/state/game.cpp, line: 457
+# File: src/gui/state/game.cpp, line: 459
+#: src/gui/qt/plugin_editor/group_tab.cpp:56
+msgid "%1 (Group does not exist)"
+msgstr "Gruppen “%1%” findes ikke."
```
Comment (sibir-ine) [inline resources/l10n/da/LC_MESSAGES/loot.po:1108]: The placeholders should match, so the translation should use `%1` in this string. Also, the context for this string is that it shows up in the **Group:** drop-down menu in the metadata editor when a plugin is assigned a group that is not defined in either the masterlist or userlist. It'd therefore be preferable to keep the same format as the original.

File: resources/l10n/da/LC_MESSAGES/loot.po [hunk 2]
```diff
@@ -4,1267 +4,2086 @@
...
-"den via hovedmenuen) for mere information."
-
 #
-# File: src/gui/html/js/translateStaticText.js, line: 392
-# File: src/gui/html/js/translateStaticText.js, line: 392
-msgid "Open Debug Log Location"
-msgstr "Åben sted for fejlsøgningslog"
+#: src/gui/query/types/get_game_data_query.h:47
+msgid "Parsing, merging and evaluating metadata..."
+msgstr "Læser, sammenfletter og evaluerer metadata…"
```
Comment (sibir-ine) [inline resources/l10n/da/LC_MESSAGES/loot.po:1563]: You type ellipses as three separate dots in most other strings, so this should probably be the same for consistency.

**Review by sibir-ine — COMMENTED** [review]
Looks good, just needs a few changes as well as mnemonics for the **Game** menu (which we discussed in Discord). Also, this isn't a big deal, but would you mind cleaning up the leftover "File:" translator comments? The gettext extractor already handles source comments. You can do so in Poedit by clicking on any strings with a comment bubble to their left & deleting their comments in the bottom-right corner.

File: resources/l10n/da/LC_MESSAGES/loot.po (hunk 1 above)
Comment (ZyPA) [reply resources/l10n/da/LC_MESSAGES/loot.po:1108]: Should be fixed in latest commit.

File: resources/l10n/da/LC_MESSAGES/loot.po (hunk 2 above)
Comment (ZyPA) [reply resources/l10n/da/LC_MESSAGES/loot.po:1563]: Resolved on discord

File: resources/l10n/da/LC_MESSAGES/loot.po [hunk 3]
```diff
@@ -375,132 +238,81 @@ msgstr "&Spil"
...
 
-#
-# File: src/gui/html/js/translateStaticText.js, line: 398
-# File: src/gui/html/js/translateStaticText.js, line: 398
 #. translators: This string is an action in the Game menu.
 #: src/gui/qt/main_window.cpp:687
-#, fuzzy
 msgid "&Fix Ambiguous Load Order"
-msgstr "Ret tvetydig indlæsningsrækkefølge"
+msgstr "&Fix tvetydig indlæsningsrækkefølge"
```
Comment (sibir-ine) [inline resources/l10n/da/LC_MESSAGES/loot.po:272]: Looks good now. Just to make sure, was changing "Ret" to "Fix" intentional?

**Review by sibir-ine — APPROVED** [review]
Thanks!

**Comment by sibir-ine** [comment]
You can also add yourself to [the credits](https://github.com/loot/loot/blob/master/docs/app/credits.rst) for Danish.

**Comment by Ortham** [comment]
Thanks 👍

**Comment by misfliction** [comment]
Thanks.

**Pull request merged**